src/index.js

```diff
@@ -3,20 +3,23 @@ import axios from "axios";
 import fs from "fs";
 import shell from "shelljs";
 
-function commits_for(contribution) {
+// Gathers needed git commands for bash to execute per provided contribution data.
+const getCommand = (contribution) => {
```

Comment:
Admittedly I used snake_case here, where camelCase is more common for JS. I should have followed your house style *pings rubber band on wrist*

Swapping the `function` for `const blah = (blah) => {}` can lead to hoisting issues. I went through a phase of `const arrow` and then reverted to `function` for most things (I use arrow functions where I want to maintain `this`, but that's not the case here)

This article touches on the [potential gotchas with arrow syntax vs function declaration](https://medium.com/the-non-traditional-developer/arrow-functions-vs-traditional-functions-in-javascript-8ff1a48ede12). I saw a better, more convincing one, but I can't find it at the moment.

`getCommand(contribution)` is still a very imperative way of expressing what you're doing. There's a disconnect between the name of the function, the parameters, and the result. The style that I use is more literate, so if you read `gitCommitCommandsFor(contribution)` it tells you more or less the same as the comment that you added. Personally, I don't think that `git` or `commands` are pulling their weight semantically here (we know by context that we're creating git commands), which is why I ended up with `commits_for(contribution) / commitsFor(contribution)`

You can also see the difference between literate and imperative at the place where the function is used:
```.map(contribution => commits_for(contribution))``` reads as map the "contribution" to the "commits for the contribution"
whereas
```.map(contribution => getCommand(contribution))``` has a disconnect; map the "contribution" to "getCommand". You can infer what it's doing (especially with the comment) but (I feel) there's an extra cognitive load.